### PR TITLE
doc update for custom store provider support connect Express-Gateway/Express-Gateway#597

### PR DESCRIPTION
**Pull request by DrMegavolt**

### CODE REVIEW

File: docs/configuration/system.config.yml/session.md [hunk 1]
```diff
@@ -9,8 +9,36 @@ OAuth 2.0 related config.
 
 Express Gateway does not need session to work. The only part that requires it is Authorization Code flow of OAuth 2.0 built in server. 
 
+Session support for OAuth2 code Grant is based on express-session npm module
```
Comment (XVincentX) [inline docs/configuration/system.config.yml/session.md:12]: Is it just for code grant or for the whole policy in general?

**Review by XVincentX — APPROVED** [review]

File: docs/configuration/system.config.yml/session.md (hunk 1 above)
Comment (DrMegavolt) [reply docs/configuration/system.config.yml/session.md:12]: for policy in general but only code exchange really requires session.

**Comment by XVincentX** [comment]
Makes sense. Let's wait @altsang to have a look into this too.

**Comment by altsang** [comment]
It was only intended for the code exchange, not the entire policy


On January 11, 2018 at 6:03:34 AM, Vincenzo Chianese (
notifications@github.com) wrote:

Makes sense. Let's wait @altsang <https://github.com/altsang> to have a
look into this too.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/ExpressGateway/express-gateway.io/pull/189#issuecomment-356942533>,
or mute the thread

**Comment by XVincentX** [comment]
Allright, then we're set!

**Comment by DrMegavolt** [comment]
The requests that are coming through gateway does not have session. 
Session is enabled **only** for oauth2 routes
And eventually we can consider disabling it . For example if you don't use code grant there is no point in it

**Review by altsang — APPROVED** [review]

**Pull request merged**